packages/platform-sdk-generator/src/model/Resource.ts

```diff
@@ -18,6 +18,6 @@ import type { Component } from "./Component.js";
 import type { StaticOperation } from "./StaticOperation.js";
 
 export interface Resource {
-  component: Component;
+  component: string;
```

Comment:
Why is this now typed as a string?

Reply:
You arent required to have a real component it turns out. I was only using this for the name anyway.